OneCommunityGlobal/HighestGoodNetworkApp
# Demi_add_id_field_to_material_props

# Description
Fixes Phase 2 WBS Bugs line item 3 “Material List”
Material items and purchase record were not having a unique ID created causing an error in the console.

## Related PRS (if any):
[PR 686
](https://github.com/OneCommunityGlobal/HGNRest/pull/686)

## Main changes explained:
- Added an _id field to the materials props

## How to test:
1. git pull
2. check into current branch
3. npm install
4. npm run start:local to run this PR locally
5. Clear site data/cache in local storage
6. log in as any user
7. go to http://localhost:3000/bmdashboard/login to confirm your login credentials
8. go to http://localhost:3000/bmdashboard/materials
9. select "View" under "Purchases"
10. Verify that you do not receive a console error for duplicate keys
11. select "View" under "Updates"
12. Verify that you do not receive a console error for duplicate keys

## Screenshots or videos of changes:
<img width="1147" alt="Fix_id" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/93569791/ea586760-87c0-48c0-8e2d-299854380283">

## Note:
None
